scripts/benchmark.sh

```diff
@@ -238,7 +262,7 @@ for server in "xline" "etcd"; do
                 printf ${FORMAT} ${QPS} ${Latency} ${Time} "${cmd}" >>${result_file}
 
             done
+            stop_all
```

Comment:
restart containers takes a long time, I recommend cleaning up the cluster inside the container

Reply:
Fixed